src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -7812,6 +7819,16 @@ public static Value group_string(
     return value;
   }
 
+  public static Value seed(ScriptRuntime controller, final Value expr) {
```

Comment:
Give params meaningful names if you can - we extract them for Typescript definitions